src/Catalog.Command.Service/Infrastructure/Data/Configurations/ProductConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Infrastructure.Data.Configurations;
+
+public class ProductConfiguration : IEntityTypeConfiguration<Product>
+{
+    public void Configure(EntityTypeBuilder<Product> builder)

```

Comment:
Please configure Price property to be money

Reply:
Done